### ISSUE TITLE
problem  about  mfsr.py

### ISSUE THREAD

**Issue by Censhengcai** [YuYin1/DA-GAN#13]
Under  .src/model/mfsr.py
the function is blow
=============解释
def mc(self, tensor): 
        # mirror and then concat
        inv_idx = torch.arange(tensor.size(0)-1, -1, -1).long().cuda()#img得到img倒序的索引   .long()  索引的记得转   不是应该在dim=w 镜像（水平翻转)吗  =======？？？？？？暂且认为是水平翻转？？？？  问作者
        inv_tensor = tensor.index_select(0, inv_idx)
        inv_tensor = tensor[inv_idx]#得到相反顺序的tenor
        return torch.cat([tensor, inv_tensor], 1)#在通道维度(dim=1)拼接：？？？岂不是两个不同的人组成不同的通道（-1,6,128,128）   不是应该在原来的水平面的吗
![QQ截图20211108111248](https://user-images.githubusercontent.com/71964196/140678636-a5a659b1-f40e-455b-b6d4-14cd7faf8fcd.png)
==============
problem：why the input of encoder was preprocessed  to be tensor with shape as（-1,6,128,128）which include two imgs from difference person，may you explain  why 。 becuase my ability of coding is poor，I  don‘t konw whether I had a incorrect。hope to reply。

**Comment by YuYin1** [comment]
Thanks for your comment. The code for the mc function is updated.

**Comment by jainam1997** [comment]
Hello everyone,
 I need help. I am a student and I got face recognition related project. I must use this repository https://github.com/YuYin1/DA-GAN. Can you please help to solve it?

Please please i really need help, I need to submit as sson as possible.

Contact me on: jainammehta1000@gmail.com
